breathecode/media/tests/urls/test_resolution_id.py

```diff
@@ -0,0 +1,179 @@
+        url = reverse_lazy('media:resolution_id', kwargs={'resolution_id': 1})
+        self.generate_models(authenticate=True)
+        data = {}
+        response = self.client.delete(url, data)
+        json = response.json()
+
+        self.assertEqual(json, {
+            'detail': "You (user: 1) don't have this capability: crud_media_resolutions for academy 1",
+            'status_code': 403
+        })
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    @patch(GOOGLE_CLOUD_PATH['client'], apply_google_cloud_client_mock())
+    @patch(GOOGLE_CLOUD_PATH['bucket'], apply_google_cloud_bucket_mock())
+    @patch(GOOGLE_CLOUD_PATH['blob'], apply_google_cloud_blob_mock())
+    def test_resolution_id_get_without_media(self):
+        """Test /answer without auth"""
+        self.headers(academy=1)
+        model = self.generate_models(authenticate=True, profile_academy=True,
+            capability='read_media_resolutions', role='potato', media_resolution=True)
+        url = reverse_lazy('media:resolution_id', kwargs={'resolution_id': 1})
+        response = self.client.get(url)
+        json = response.json()
+
+        self.assertEqual(json, {'detail': 'resolution-media-not-found', 'status_code': 404})
+        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+        self.assertEqual(self.all_media_resolution_dict(), [])
+
+    @patch(GOOGLE_CLOUD_PATH['client'], apply_google_cloud_client_mock())
+    @patch(GOOGLE_CLOUD_PATH['bucket'], apply_google_cloud_bucket_mock())
+    @patch(GOOGLE_CLOUD_PATH['blob'], apply_google_cloud_blob_mock())
+    def test_info_id_resolution_get_with_id(self):
+        """Test /info/media:id/resolution"""
+        self.headers(academy=1)
+        model = self.generate_models(authenticate=True, media_resolution=True, media=True,
+            capability='read_media_resolutions', role='potato', profile_academy=True)
+        model_dict = self.remove_dinamics_fields(model['media_resolution'].__dict__)
+        url = reverse_lazy('media:resolution_id', kwargs={'resolution_id': 1})
+        response = self.client.get(url)
+        json = response.json()
+        expected = {
+            'id': model['media_resolution'].id,
+            'hash': model['media'].hash,
+            'width': model['media_resolution'].width,
+            'height': model['media_resolution'].height,
+            'hits': model['media_resolution'].hits,
+        }
+
+        self.assertEqual(json, expected)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(self.count_media_resolution(), 1)
+        self.assertEqual(self.get_media_resolution_dict(1), model_dict)
+
+    @patch(GOOGLE_CLOUD_PATH['client'], apply_google_cloud_client_mock())
+    @patch(GOOGLE_CLOUD_PATH['bucket'], apply_google_cloud_bucket_mock())
+    @patch(GOOGLE_CLOUD_PATH['blob'], apply_google_cloud_blob_mock())
+    def test_resolution_id_delete_with_different_academy(self):
```

Comment:
This test actually can be removed because we can remove a resolution of other academy